templates/details/resources.html

```diff
@@ -114,23 +118,25 @@ <h3 class="p-heading--5">Release history</h3>
     panelToggleButtons.forEach(function(panelToggleButton) {
       panelToggleButton.addEventListener("click", function(e) {
         var target = e.target;
-        var panel = panelToggleButton.querySelector(".p-inline-list__panel");
+        var panel = document.getElementById(
+          target.getAttribute("aria-controls")
+        );
 
-        if (target !== panel) {
-          panelToggleButton.classList.toggle("is-open");
-        }
+        target.classList.toggle("is-active");
+        panel.classList.toggle("is-open");
```

Comment:
Two comments:
- Only click events are being handled, so there's no way to close it without clicking; it stays open even if you tab to another part of the page. We should at least handle pressing the escape key to close the dropdown and return focus to the button.
- I think it'd be nice to focus on the input when it's revealed, since it automatically selects the value, making it even easier to copy. It highlights the whole lot in orange though, so might need a design +1.